g4py.sh

```diff
@@ -0,0 +1,51 @@
+package: G4PY
+version: "%(tag_basename)s%(defaults_upper)s"
+source: https://github.com/alisw/geant4
+tag: v4.10.01.p03
+requires:
+  - GEANT4
+  - Python-modules
+  - ROOT
+  - boost
+  - XercesC
+  - opengl
+build_requires:
+  - CMake
+  - "Xcode:(osx.*)"
+env:
+  G4PYINSTALL: "$G4PY_ROOT"
+---
+#!/bin/bash -e
+cp -a $SOURCEDIR/environments/g4py/* $INSTALLROOT
```

Comment:
I have tested your suggestion and it did not work. It is still installing into the directory of the source files, now located in `$BUILDDIR`. 

I have checked the source code and am now fully convinced, that this is working as intended:
`# installation prefixes for libraries
set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
`
In an excerpt from the build log the cmake command is as follows:
`cmake . -DCMAKE_BUILD_TYPE=RELWITHDEBINFO -DBoost_NO_SYSTEM_PATHS=TRUE -DCMAKE_INSTALL_PREFIX=/opt/SHiP/alibuild/sw/INSTALLROOT/f3c690189ff37e165859e5aec89962d6a2c1bf8b/ubuntu1604_x86-64/G4PY/v10.3.1-ship2-2 -DBOOST_ROOT=/opt/SHiP/alibuild/sw/ubuntu1604_x86-64/boost/v1.64.0-alice1-1 -DXERCESC_ROOT_DIR=/opt/SHiP/alibuild/sw/ubuntu1604_x86-64/XercesC/v3.1.4-2 -DBoost_NO_BOOST_CMAKE=TRUE`
While a few lines later the following is logged:
`CMAKE_INSTALL_PREFIX: /opt/SHiP/alibuild/sw/BUILD/f3c690189ff37e165859e5aec89962d6a2c1bf8b/G4PY`
Furthermore nearly all python files are installed in either source/ or site-modules/ in `$BUILDDIR.`

As this is clearly intended behaviour, I would stick with the original idea of copying the source files to '$INSTALLDIR` and building inside `$BUILDDIR`. It would probably be a good idea to use rsync for this, if only for consistency